packages/sdk/src/types/common.ts

```diff
@@ -73,12 +73,6 @@ export interface SignTransction{
   statusCallback?: (result: ISubmittableResult) => void;
 }
 
-export interface FetchRoles{
-  id: string;
-  name: string;
-  enabled: boolean;
-}
-
 export interface ResponseFetchGroup{
```

Comment:
Need some clarification here. fetchRoles(), fetchGroups(), and fetchPermissions() all return the same object which contains an id, name, and enabled. Should a unique return object be created here to differentiate between the functions?

For example rather than just having ResponseFetchGroup[] return for all of them, we could add ResponseFetchRole and ResponseFetchPermission as well.

Reply:
Change the object name to a more generic name:
`FetchResponseData`